packages/aws-appsync/package.json

```diff
@@ -19,7 +19,7 @@
     "test-watch": "jest --watch"
   },
   "dependencies": {
-    "@redux-offline/redux-offline": "2.5.2-native.0",
+    "@redux-offline/redux-offline": "2.5.2-native.3",
```

Comment:
We should add '@react-native-community/async-storage' to peer dependencies and docs as well.